File: Daily Challanges/2026-02-February/2026-02-20.js
function isValidTrick(trickName) {
  const validFirsts = [
    "Misty",
    "Ghost",
    "Thunder",
    "Solar",
    "Sky",
    "Phantom",
    "Frozen",
    "Polar"
  ];

  const validSeconds = [
    "Twister",
    "Icequake",
    "Avalanche",
    "Vortex",
    "Snowstorm",
    "Frostbite",
    "Blizzard",
    "Shadow"
  ];
  const [word1, word2] = trickName.split(' ')
  return validFirsts.includes(word1) && validSeconds.includes(word2);
}


const test1 = isValidTrick("Polar Vortex");
const test2 = isValidTrick("Solar Icequake");
const test3 = isValidTrick("Thunder Blizzard");
const test4 = isValidTrick("Phantom Frostbite");
const test5 = isValidTrick("Ghost Avalanche");
const test6 = isValidTrick("Snowstorm Shadow");
const test7 = isValidTrick("Solar Sky");

const check1 = true;
const check2 = true;
const check3 = true;
const check4 = true;
const check5 = true;
const check6 = false;
const check7 = false;

console.log(`Test1: ${test1 === check1}`);
console.log(`Test2: ${test2 === check2}`);
console.log(`Test3: ${test3 === check3}`);
console.log(`Test4: ${test4 === check4}`);
console.log(`Test5: ${test5 === check5}`);
console.log(`Test6: ${test6 === check6}`);
console.log(`Test7: ${test7 === check7}`);

File: Daily Challanges/2026-02-February/2026-02-20.py
def is_valid_trick(trick_name):
    valid_firsts = [
        "Misty",
        "Ghost",
        "Thunder",
        "Solar",
        "Sky",
        "Phantom",
        "Frozen",
        "Polar"
    ];

    valid_seconds = [
        "Twister",
        "Icequake",
        "Avalanche",
        "Vortex",
        "Snowstorm",
        "Frostbite",
        "Blizzard",
        "Shadow"
    ];

    word1, word2 = trick_name.split(' ')
    return word1 in valid_firsts and word2 in valid_seconds

test1 = is_valid_trick("Polar Vortex")
test2 = is_valid_trick("Solar Icequake")
test3 = is_valid_trick("Thunder Blizzard")
test4 = is_valid_trick("Phantom Frostbite")
test5 = is_valid_trick("Ghost Avalanche")
test6 = is_valid_trick("Snowstorm Shadow")
test7 = is_valid_trick("Solar Sky")

check1 = True
check2 = True
check3 = True
check4 = True
check5 = True
check6 = False
check7 = False

print(f"Test1: {test1 == check1}")
print(f"Test2: {test2 == check2}")
print(f"Test3: {test3 == check3}")
print(f"Test4: {test4 == check4}")
print(f"Test5: {test5 == check5}")
print(f"Test6: {test6 == check6}")
print(f"Test7: {test7 == check7}")